src/content/defi/index.md

```diff
@@ -0,0 +1,245 @@
+
+<DocLink to="/wallets/" title="More on wallets" />
+
+#### Stream money around the globe... {#stream-money}
+
+It's also possible to stream money over Ethereum. This would let you pay someone their salary by the second, giving them access to their money whenever they need it.
+
+### Access to stable currencies {#stablecoins}
+
+Although a lot of cryptocurrencies are known for their volatility, the defi community has invested a lot of time in creating stability out of volatility with stablecoins. Their value stays pegged to an another asset, usually a popular currency like dollars.
+
+Coins like Dai or USDC have a value that stays within a few cents of a dollar. This makes them perfect for earning or retail. So far many people in Latin America have used stablecoins as a way of protecting their savings in a time of great uncertainty with their government-issued currencies.
+
+<DocLink to="/stablecoins/" title="More on stablecoins" />
+
+### Lending and borrowing {#lending}
+
+Today, lending and borrowing money all revolves around the individuals involved. Banks need to know whether you're likely to repay a loan before lending. Some see this as an invasion of privacy.
+
+Decentralized lending works without either party having to identify themselves. Instead the borrower must put up collateral that the lender will automatically receive if their loan is not repaid.
+
+Loans can be peer-to-peer, meaning a borrower will borrow directly from a specific lender. Or they can be pool-based where lenders provide liquidity to a pool that borrowers can borrow from.
+
+Some lenders even accept NFTs as collateral. NFTs are a deed to a unique asset, like a painting. [More on NFTs](/nfts/)
+
+A more experimental form of decentralized lending lets you borrow without collateral or providing any personal information...
+
+#### Flash loans {#flash-loans}
+
+Flash loans are not advised if you're not a technical expert but they hint at what might be possible to everyone in the future. With Ethereum you can take out huge loans without putting up any collateral whatsoever.
+
+It works on the premise that the loan is taken out and paid back within the same transaction. If it can't be paid back, the transaction reverts as if nothing ever happened.
+
+This means a lot of logic must be included in a very bespoke transaction. A simple example might be someone using a flash loan to borrow as much of an asset at one price so they can sell it on a different exchange where the price is higher.
+
+So in a single transaction the following happens:
+
+- You borrow X amount of $asset at $1.00 from exchange A
+- You sell X $asset on exchange B for $1.10
+- You pay back loan to exchange A
+- You keep the profit minus the transaction fee
+
+If exchange B's supply dropped suddenly and the user wasn't able to buy enoughh to cover the original loan, the transaction would simply fail.
+
+### Start saving with crypto {#saving}
```

Comment:
Since we have a "saving" section here (which is the same as lending), perhaps we should update the section above to just "borrowing". Flash loans are about borrowing & your description of lending/borrowing is mostly about borrowing.